src/chart/builder.rs

```diff
@@ -18,7 +18,8 @@ pub enum LabelAreaPosition {
 /// With the hlep of this object, we can convert a basic drawing area into a chart context, which
 /// allows the high-level chartting API beening used on the drawing area.
 pub struct ChartBuilder<'a, 'b, DB: DrawingBackend> {
-    label_area_size: [u32; 4], // [upper, lower, left, right]
+    label_area_size: [i32; 4], // [upper, lower, left, right]
+    label_area_inset: [bool; 4],
```

Comment:
Just want to discuss with you. Since what I am planning to do is allow developer give a negative label area size, and it will automatically inverts the direction of the labels. Is there any particular reason for explicitly setting this ?

Reply:
That was first option i wanted to implement, but then i realized that negative width is usefult itself. It gives more control over drawing areas. It allows to overlay one drawing area onto one with negative size.